src/Sulu/Bundle/MediaBundle/Tests/Features/collection_management.feature

```diff
@@ -16,16 +19,17 @@ Feature: Collection management
 
     Scenario: Edit collection
         Given the media collection "Foobar" exists
-        And I am on "/admin/#media/collections"
-        And I expect a thumbnail to appear
-        When I click on the element ".thumbnail"
+        And I am on "/admin/#media/collections/root"
+        And I expect a data-navigation to appear
+        When I click on the element ".data-navigation-item"
         And I expect the "husky.loader.initialized" event
         Then I should see "Foobar"
```

Comment:
Just a note, this should look more like:

````cucumber
Given the media collection "Foobar" exists
And I am on " ... "
And I wait for a data-navigation to apprear
When I click on the element " ... "
And wait for the "husky.loader.initialized" event
Then I should see "Foobar"
````
So `Given` is the background, `When` explains actions which you take, and `Then` is the result of the actions.